pages/game.js

```diff
@@ -18,6 +18,16 @@ import {
 } from '../src/core/game';
 import { shuffle } from '../src/core/shuffler';
 
+export const title = (isLoading, isVictory, turn) => {
```

Comment:
Is there any reason you export it ? It doesn't seem tested or used anywhere.

Reply:
I've added more tests